www/js/index.js

```diff
@@ -136,19 +127,33 @@ function findLine(e) {
     lineFound.click();
   }
 }
-
 // Keyboard shortcuts
 if (typeof Mousetrap !== 'undefined') {
   Mousetrap.bindGlobal('esc', escKey);
 
-  Mousetrap.bindGlobal(['command+1', 'ctrl+1'], waheguruSlide);
-  Mousetrap.bindGlobal(['command+2', 'ctrl+2'], moolMantraSlide);
-  Mousetrap.bindGlobal(['command+3', 'ctrl+3'], emptySlide);
-  Mousetrap.bindGlobal(['command+4', 'ctrl+4'], anandSahibBhog);
-  Mousetrap.bindGlobal(['command+5', 'ctrl+5'], helpGuideShortcut);
-  Mousetrap.bindGlobal(['command+6', 'ctrl+6'], legendShortcut);
-
-  Mousetrap.bindGlobal(['command+/', 'ctrl+/'], searchBarShortcut);
+  Mousetrap.bindGlobal(['fn+1'], slideShortcuts.waheguru);
+  Mousetrap.bindGlobal(['fn+2'], slideShortcuts.moolMantra);
+  Mousetrap.bindGlobal(['fn+3'], slideShortcuts.empty);
+
+  Mousetrap.bindGlobal(['fn+4'], ceremonyShortcuts.anandSahibBhog);
+
+  Mousetrap.bindGlobal('fn+5', interfaceShortcuts.help);
+  Mousetrap.bindGlobal('fn+6', interfaceShortcuts.legend);
+  Mousetrap.bindGlobal('mod+/', interfaceShortcuts.searchBar);
+
+  for (let g = 0; g <= dhanGuruText.length - 1; g += 1) {
+    if (g <= 8) {
+      Mousetrap.bindGlobal([`mod+${g + 1}`], () =>
+        global.controller.sendText(dhanGuruText[g], true),
+      );
+      // eslint-disable-next-line eqeqeq
+    } else if (g == 9) {
+      Mousetrap.bindGlobal(['mod+0'], () => global.controller.sendText(dhanGuruText[g], true));
+      // eslint-disable-next-line eqeqeq
```

Comment:
remove this and use `===`  See explanation: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Equality_comparisons_and_sameness